cmd/cluster-image-registry-operator/main.go

```diff
@@ -17,34 +16,34 @@ import (
 )
 
 func printVersion() {
-	glog.Infof("Cluster Image Registry Operator Version: %s", version.Version)
-	glog.Infof("Go Version: %s", runtime.Version())
-	glog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
+	klog.Infof("Cluster Image Registry Operator Version: %s", version.Version)
+	klog.Infof("Go Version: %s", runtime.Version())
+	klog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
 }
 
 func main() {
-	flag.Parse()
-	flag.Lookup("logtostderr").Value.Set("true")
+	klog.InitFlags(nil)
+	defer klog.Flush()
```

Comment:
It seems the operator ignores flags. You can try it with

`make build && WATCH_NAMESPACE=foo ./tmp/_output/bin/cluster-image-registry-operator -skip_headers`

and the operator should stop use prefixes like `I0712 16:45:17.551673   28281 main.go:20]`